wpengine/faustjs
# Move toolbar exports into their own subdirectory

## Tasks


## Description

These changes move Toolbar entrypoints from `@faustwp/core` to  `@faustwp/core/toolbar` in an effort to clean up the main entrypoints file.

## Testing

1. Clone this feature branch: `MERL-752-toolbar-entrypoints`
2. Run `npm run build`
3. Run `npm run dev -w @faustwp/getting-started-example`
4. Notice that the custom toolbar nodes from the example project are loading with no errors.
<img width="665" alt="Screen Shot 2023-02-15 at 3 56 46 PM" src="https://user-images.githubusercontent.com/6676674/219157624-54428bbd-3940-48d9-bd03-0c2bb8fddda5.png">

## Dependent PRs

- [x] https://github.com/wpengine/faustjs/pull/1272
